src/main/java/org/broadinstitute/hellbender/tools/walkers/realignmentfilter/RealignmentEngine.java

```diff
@@ -132,8 +132,9 @@ private final static boolean mightSupportInsertion(final CigarElement cigarEleme
         return new ArrayList<>(commonAlignments);
     }
 
+    // note the conversion from exclusive end (BWAMemAlignment) to inclusive end (SimpleInterval)
     private static SimpleInterval convertToInterval(final BwaMemAlignment alignment) {
-        return new SimpleInterval(Integer.toString(alignment.getRefId()), alignment.getRefStart(), alignment.getRefEnd());
+        return new SimpleInterval(Integer.toString(alignment.getRefId()), alignment.getRefStart(), alignment.getRefEnd() - 1);
```

Comment:
I think we need to go the other way don't we?  Increase the start and leave the end alone?

Reply:
Suppose the last included position is 5.

1) Account for exclusive --> inclusive stop.
BWA mem alignment is exclusive, hence its end is 6.  SimpleInterval is inclusive, so its end should be 5, which is BWA's end - 1.

2) account for 0-based --> 1-based contig
Add 1 to everything, so that adding 1 to the end cancels subtracting 1, while the start is incremented by 1.

@lbergelson You're right.  Thank you!

Reply:
Sounds good to me!